modules/component/Makefile

```diff
@@ -46,6 +56,20 @@ component/build: %build: %init
 	${COMPONENT_BUILD_COMMAND} ${COMPONENT_DOCKER_REPO}/${COMPONENT_NAME}:${COMPONENT_VERSION}${COMPONENT_TAG_EXTENSION}
 	@echo "Built the following image: ${COMPONENT_DOCKER_REPO}/${COMPONENT_NAME}:${COMPONENT_VERSION}${COMPONENT_TAG_EXTENSION}"
 
+.PHONY: component/build-coverage
+## Build the component
+component/build-coverage: %build-coverage: %init %build
```

Comment:
If you're going to prereq %build, that target already calls (prereqs) %init

Reply:
ok